File: TradeItIosEmsApi/TradeItOAuthLoginPopupUrlForMobileRequest.m
#import "TradeItOAuthLoginPopupUrlForMobileRequest.h"

@implementation TradeItOAuthLoginPopupUrlForMobileRequest

- (id)initWithApiKey:(NSString *)apiKey
              broker:(NSString *)broker
interAppAddressCallback:(NSString *)interAppAddressCallback {
    self = [super init];

    if (self) {
        self.apiKey = apiKey;
        self.broker = broker;
        self.interAppAddressCallback = interAppAddressCallback;
    }

    return self;
}

@end

File: TradeItIosEmsApi/TradeItOAuthLoginPopupUrlForMobileRequest.swift
class TradeItOAuthLoginPopupUrlForMobileRequest: TradeItRequest {
    var apiKey: String
    var broker: String
    var interAppAddressCallback: String
    
    init(apiKey: String, broker: String, interAppAddressCallback: String) {
        self.apiKey = apiKey
        self.broker = broker
        self.interAppAddressCallback = interAppAddressCallback
    }
}
